lib/Mojolicious/resources/templates/mojo/menubar.html.ep

```diff
@@ -86,6 +150,11 @@
       $('html, body').animate({scrollTop: offset.top - mojobarHeight}, 1);
     }
   }
+  document.querySelector("#mojobar-brand").addEventListener("click", function (e) {
```

Comment:
The rest of the file uses jQuery for everything, shouldn't it be consistent?

Reply:
I agree it should be consistent. If you want, I'll make a new PR later on where we avoid loading jQuery, but for now I'll update this to use jQuery.